### PR TITLE
feat(FAQ): New in-wallet faq page

### PR DESCRIPTION
**Pull request by jstettner**

### CODE REVIEW

File: assets/css/modules/_faq.scss [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  hr {
+    width: 100%;
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #5f5f5f;
+    margin: 1em 0;
+    padding: 0;
+    opacity: 0.5;
+  }
```
Comment (plondon) [inline assets/css/modules/_faq.scss]: is this just a border bottom?

File: assets/css/modules/_faq.scss [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  
+  h2 {
+    margin: 0;
+    font-size: 23px;
+    font-weight: 300;
+  }
+  
+  .faq-box {
+    margin-top: 61px;
+  }
```
Comment (plondon) [inline assets/css/modules/_faq.scss]: lets avoid random numbers here

**Comment by jstettner** [comment]
@plondon I fixed those problems

**Comment by jstettner** [comment]
faq

**Pull request merged**